### PR TITLE
Added CLI option and API methods for min priority fee

### LINKED ISSUE #1434: Document the JSON-RPC methods: `miner_setMinPriorityFee` and `miner_getMinPriorityFee`
Document the new JSON-RPC methods: `miner_setMinPriorityFee` and `miner_getMinPriorityFee`.

See PR: [Implement miner_setMinPriorityFee and miner_getMinPriorityFee #6080](https://github.com/hyperledger/besu/pull/6080) for details.

### PR DESCRIPTION
**Pull request by MadelineMurray**
Fixes #1434

### CODE REVIEW

**Comment by MadelineMurray** [comment]
Also fixes #1438

**Review by fab-10 — APPROVED** [review]

File: docs/public-networks/reference/api/index.md [hunk 1]
```diff
@@ -5459,6 +5463,49 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"miner_changeTargetGasLimit","par
 
 <!--/tabs-->
 
+### `miner_getMinPriorityFee`
+
+Gets the minimum priority fee per gas (in Wei) offered by a transaction offers to be included in a block. The initial value is set using the [`--min-priority-fee`](../cli/options.md#min-priority-fee) command line option, or defaults to 0, if the command line option is not specified.
```
Comment (alexandratran) [inline docs/public-networks/reference/api/index.md:5468]:
```suggestion
Gets the minimum priority fee per gas (in Wei) offered by a transaction to be included in a block. The initial value is set using the [`--min-priority-fee`](../cli/options.md#min-priority-fee) command line option, or is set to `0` if the command line option is not specified.
```

File: docs/public-networks/reference/api/index.md [hunk 2]
```diff
@@ -5459,6 +5463,49 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"miner_changeTargetGasLimit","par
...
+Use [`miner_setMinPriorityFee`](#minersetminpriorityfee) to change the current value of the fee.
+
+#### Parameters
+
+None
+
+#### Returns
+
+`result`: _string_ - Minimum priority fee per gas (in Wei) in hexadecimal
```
Comment (alexandratran) [inline docs/public-networks/reference/api/index.md:5477]:
```suggestion
`result`: _string_ - Minimum priority fee per gas (in Wei) as a hexadecimal string
```

File: docs/public-networks/reference/api/index.md [hunk 3]
```diff
@@ -5508,6 +5555,50 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"miner_setCoinbase","params":["0x
 
 <!--/tabs-->
 
+### `miner_setMinPriorityFee`
+
+Sets the minimum priority fee per gas (in Wei) offered by a transaction offers to be included in a block. The initial value is set using the [`--min-priority-fee`](../cli/options.md#min-priority-fee) command line option, or defaults to 0, if the command line option is not specified.
```
Comment (alexandratran) [inline docs/public-networks/reference/api/index.md:5560]:
```suggestion
Sets the minimum priority fee per gas (in Wei) offered by a transaction to be included in a block. The initial value is set using the [`--min-priority-fee`](../cli/options.md#min-priority-fee) command line option, or is set to `0` if the command line option is not specified.
```

File: docs/public-networks/reference/cli/options.md [hunk 4]
```diff
@@ -1873,6 +1873,41 @@ In a [free gas network](../../../private-networks/how-to/configure/free-gas.md),
...
+
+# Configuration file
+
+```bash
+min-priority-fee=7
+```
+
+<!--/tabs-->
+
+The minimum priority fee per gas (in Wei) offered by a transaction offers to be included in a block. The default is 0 Wei.
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:1906]:
```suggestion
The minimum priority fee per gas (in Wei) offered by a transaction to be included in a block. The default is `0`.
```

File: docs/public-networks/reference/cli/options.md [hunk 5]
```diff
@@ -1873,6 +1873,41 @@ In a [free gas network](../../../private-networks/how-to/configure/free-gas.md),
...
+For a running node, use: 
+* [`miner_getMinPriorityFee`](../api/index.md#minergetminpriorityfee) to get the value
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:1908]:
```suggestion
* [`miner_getMinPriorityFee`](../api/index.md#minergetminpriorityfee) to get the value.
```

**Review by alexandratran — APPROVED** [review]
Some edits, lgtm.

File: docs/public-networks/reference/api/index.md [hunk 6]
```diff
@@ -5408,7 +5408,11 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"eth_uninstallFilter","params":["
 
 ## `MINER` methods
 
-The `MINER` API methods allow you to control the node’s mining operation.
+The `MINER` API methods allow you to control: 
+
+* The node’s mining operation.
```
Comment (mjsmike62) [inline docs/public-networks/reference/api/index.md:5413]: "The" article not needed. 

Usually, inanimate objects don't take a possessive.

**Review by mjsmike62 — COMMENTED** [review]
One small, suggested edit. But LGTM.

**Pull request merged**